src/libs/OptionsListUtils.js

```diff
@@ -125,15 +125,11 @@ Onyx.connect({
  * @returns {Boolean}
  */
 function isSearchStringMatch(searchValue, searchText) {
-    const matchRegexes = [
-        new RegExp(`^${Str.escapeForRegExp(searchValue)}$`, 'i'),
-        new RegExp(`^${Str.escapeForRegExp(searchValue)}`, 'i'),
-        new RegExp(Str.escapeForRegExp(searchValue), 'i'),
```

Comment:
NAB: I'm not a regex expert so I'm just curious. Since we're searching by words now,  these old regex patterns are just redundant right? My guess was that they're just looking for the word within the begging, middle, and end of the expression.

Reply:
Yes, these regex expressions were used to search the value in the beginning, middle and end of the string. It was important for sorting results based on the place of the input value. Now, we are ordering results by last message time, so these regex patterns are redundant.